testing/steps/test_step_cloner_for_each_data_input.py

```diff
@@ -13,29 +16,68 @@
 })
 
 
-class SomeStepInverseTransform(SomeStep):
-    def fit_transform(self, data_inputs, expected_outputs=None):
-        return self, 'fit_transform'
+def test_step_cloner_should_transform():
+    tape = TapeCallbackFunction()
+    p = Pipeline([
+        StepClonerForEachDataInput(
+            Pipeline([
+                FitCallbackStep(tape),
+                MultiplyByN(2)
+            ])
+        )
+    ])
+    data_inputs = _create_data((2, 2))
+
+    processed_outputs = p.transform(data_inputs)
+
+    assert isinstance(p[0].steps[0], Pipeline)
+    assert isinstance(p[0].steps[1], Pipeline)
+    assert np.array_equal(processed_outputs, data_inputs * 2)
+
+
+def test_step_cloner_should_fit_transform():
+    tape = TapeCallbackFunction()
+    p = Pipeline([
+        StepClonerForEachDataInput(
```

Comment:
do we rlly need pipeline wrapper here? i think not (same as previous 2 comments0.